Wikipedia/Code/ReadingThemesControlsViewController.swift

```diff
@@ -109,41 +118,37 @@ open class WMFReadingThemesControlsViewController: UIViewController {
         }
     }
     
-    @IBAction func standardThemeButtonPressed(_ sender: Any) {
-        let theme = ["theme": Theme.standard]
-        
-        applyBorder(to: standardThemeButton)
-        removeBorderFrom(lightThemeButton)
-        removeBorderFrom(darkThemeButton)
-        
-        NotificationCenter.default.post(name: Notification.Name(WMFReadingThemesControlsViewController.WMFUserDidSelectThemeNotification), object: nil, userInfo: theme)
-    }
-    
-    @IBAction func lightThemeButtonPressed(_ sender: Any) {
-        let theme = ["theme": Theme.light]
+    @IBAction func themeButtonPressed(_ sender: UIButton) {
```

Comment:
I made a similar adjustment when adding the sepia theme https://github.com/wikimedia/wikipedia-ios/pull/1627/files#diff-f1a8c1fbb0db56f1a452a1edb51af148 - I can work through the merge conflict tomorrow morning